source/docs/appendices/release-process.md

```diff
@@ -0,0 +1,20 @@
+---
+id: release-process
+title: Release process
+---
+
+Front-Commerce is built around the motto "Make it work, then make it better". This means that it is possible **today** to develop and deploy your website to production. But we still have a lot of things to work on to accomplish our mission: make your life easier, and your customers happier.
```

Comment:
The "But we still have a lot of things to work on" feels a bit negative. What's in FC today is already great for external developers. This phrasing is a bit downsizing what we can provide.